package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Base ESLint styles by Underdog.io",
   "main": "index.js",
   "scripts": {
-    "lint": "eslint -c ./index.js ./**/*.js",
+    "lint": "eslint -c ./index.js ./test/index.js && eslint -c ./es6.js ./test/es6.js",
```

Comment:
We probably still want to test `index.js` and `es6.js` under the ES5 rules (`index.js` config).

We can also break this up into multiple scripts, e.g.

```json
{
  "lint": "npm run lint-es5 && npm run lint-es6",
  "lint-es5": "eslint -c ./index.js ./index.js ./test/index.js",
  "lint-es6": "eslint -c ./es6.js ./test/es6.js"
}
```